impersonate/urls.py

```diff
@@ -1,19 +1,31 @@
-from django.conf.urls import patterns, url
+from django.conf.urls import url
 
-urlpatterns = patterns(
-    'impersonate.views',
+from .views import impersonate, list_users, search_users, stop_impersonate
```

Comment:
they're being referenced directly on lines 13, 16, 20, 24 now instead of the strings. do you have a preferred other way to do it?